exporter/clickhouseexporter/exporter_metrics.go

```diff
@@ -0,0 +1,109 @@
+// Copyright  The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package clickhouseexporter // import "github.com/open-telemetry/opentelemetry-collector-contrib/exporter/clickhouseexporter"
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"go.opentelemetry.io/collector/pdata/pmetric"
+	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/exporter/clickhouseexporter/internal"
+)
+
+type metricsExporter struct {
+	client *sql.DB
+
+	logger *zap.Logger
+	cfg    *Config
+}
+
+func newMetricsExporter(logger *zap.Logger, cfg *Config) (*metricsExporter, error) {
+	if err := cfg.Validate(); err != nil {
+		return nil, err
+	}
+	if err := createDatabase(cfg); err != nil {
```

Comment:
Would it be possible to move this code to a start method, that runs as part of the exporter lifecycle?

Reply:
The function `newMetricsExporter()` is follow the style of `newTracesExporter()`, I think we can use another PR to deal with them.

Reply:
ok,here https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/17558